### PR TITLE
Symplify the network config

### PR DESCRIPTION
**Pull request by pkeroulas**
Let docker assign IPs.

### CODE REVIEW

**Comment by pkeroulas** [comment]
I admit the benefit/risk ratio of the range assignment is quite limited for external interfaces.
However, I would it still remove the IP from `private-nmos` since there is no such risk of conflict in the internal network.

File: docker-compose.yml [hunk 1]
```diff
@@ -58,20 +56,15 @@ services:
     networks:
 # Temporary WA - Force container to only have a single interface so mDNS advertisements test correctly - See: https://github.com/AMWA-TV/nmos-testing/issues/214
 #     private-nmos:
-#       ipv4_address: '172.30.101.5'
       external:
-        ipv4_address: '192.168.6.103'
+        ipv4_address: '192.168.1.103'
```
Comment (garethsb-sony) [inline docker-compose.yml:60]: Typo, @pkeroulas ??

Comment (rhastie) [reply docker-compose.yml:60]: My intention is to remove the private network entirely... Hopefully you'll see a commit later today

**Comment by rhastie** [comment]
The backend network has now been removed on the master branch - This should make this PR redundant now...

**Pull request closed without merging**